### ISSUE TITLE
firebase queue support

### ISSUE THREAD

**Issue by muhammadnasr** [NozeIO/Noze.io#3]
nice work
do you plan to support firebase queue?
https://github.com/firebase/firebase-queue

**Comment by helje5** [comment]
No, not really. I mean, we are perfectly open to contributions, but I don't know a single person using or wanting to use that :-) I'll keep this open as something to consider.
